egs2/dirha_wsj/asr1/run.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+# Set bash to 'debug' mode, it will exit on :
+# -e 'error', -u 'undefined variable', -o ... 'error in pipeline', -x 'print commands',
+set -e
+set -u
+set -o pipefail
+
+mic=L1C     # Beam_Circular_Array Beam_Linear_Array KA6 L1C
+
+local_data_opts="--mic ${mic}"
+
+stage=1
+stop_stage=11
+nj=16
+decode_asr_model=valid.loss.best.pth 
+
```

Comment:
Remove these four parameters from run.sh
stage, stop_stage, nj, decode_asr_model because these are not task depended.

Reply:
Done.